debian/rules

```diff
@@ -116,9 +116,9 @@ install: build
 	install -m0644 $(CURDIR)/debian/ifup@.service $(CURDIR)/debian/packetfence/etc/systemd/system
 #endif
 	install -d -m0700 $(CURDIR)/debian/packetfence/etc/default
-	# copy the rsyslog configuration file 
-	install -d -m0755 $(CURDIR)/debian/packetfence/etc/rsyslog.d
-	install -m0600 $(CURDIR)/debian/packetfence.rsyslog $(CURDIR)/debian/packetfence/etc/rsyslog.d/packetfence.conf
+	# copy the rsyslog systemd drop in
+	install -d -m0755 $(CURDIR)/debian/packetfence/etc/systemd/system/rsyslog.service.d/
+	install -m0600 $(CURDIR)/packetfence.rsyslog-drop-in.service  $(CURDIR)/debian/packetfence/etc/systemd/system/rsyslog.service.d/packetfence.conf
```

Comment:
packetfence.rsyslog-drop-in.service is missing

Reply:
Fixed